zebrad/tests/common/lightwalletd/rpc.rs

```diff
@@ -0,0 +1,76 @@
+//! Lightwalletd gRPC interface and utility functions.
```

Comment:
Optional: I wonder if this module name will get confused with node JSON-RPC?

Would `wallet_grpc` be better?